app/services/create_performance_json.service.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+const moment = require('moment')
+const lodash = require('lodash')
+
+const filteredServices = require('./get_filtered_services.service')
+const performanceDataService = require('./get_performance_data.service')
+
+async function createPerformanceJson() {
+  const fromDate = moment.utc().subtract(7, 'days').startOf('day').format()
+  const toDate = moment().utc().startOf('day').format()
+
+  const services = await filteredServices.fetchAndFilterServices()
+  const performanceReport = await performanceDataService.getPerformanceReport(fromDate, toDate)
+
+  const countBySector = {}
+  const countByOrganisation = {}
+
+  services.forEach(service => {
+    if (service.sector) {
+      const sector = service.sector
+      lodash.set(countBySector, sector, countBySector[sector] + 1 || 1)
+    }
+    const name = service.serviceName
```

Comment:
This should be the `merchant_details.name` from the service response from adminusers, looks like we're getting the `service_name` which is an object containing service names in different languages.